packages/components/src/List/ListComposition/VList/VList.component.js

```diff
@@ -8,7 +8,8 @@ function VList(props) {
 	return <VirtualizedList collection={collection} type={displayMode.toUpperCase()} {...props} />;
 }
 
-VList.Content = VirtualizedList.Content;
-VList.propTypes = VirtualizedList.propTypes;
-
+Object.entries(VirtualizedList).forEach(([key, value]) => {
```

Comment:
Could you add a little comment here to explain that is used to reference the columns compund api.
Seems weird when you don't know what's going on.